spec/swagger_helper.rb

```diff
@@ -698,6 +698,74 @@
               amount: { "$ref" => "#/components/schemas/currency" },
             },
           },
+          SelfEmploymentType: {
+            type: :string,
+            enum: %i[partnership company_director sole_trader],
+          },
+          SelfEmployment: {
+            type: :object,
+            required: %i[name self_employment_type net_profit_or_drawings percentage_owned business_capital],
+            additionalProperties: false,
+            properties: {
+              name: { type: :string },
+              self_employment_type: { "$ref" => "#/components/schemas/SelfEmploymentType" },
+              net_profit_or_drawings: {
```

Comment:
what about income tax and NI?

Reply:
@davidread I'm assuming that income tax and national insurance i will be submitted with 'employment income' (along with benefits_in_kind). The employers NI doesn't look like it comes into the equation, its all subsumed into this 'net_profit' figure